apps/backoffice/src/components/services/service-context-menu.tsx

```diff
@@ -210,6 +213,20 @@ export const ServiceContextMenu = ({
     </Tooltip>
   );
 
+  /** Show App Preview action button */
+  const renderAppPreviewAction = () => (
+    <Tooltip title={t("service.actions.preview")} placement="top" arrow>
+      <Button
+        size="medium"
+        variant="text"
+        sx={{ bgcolor: "background.paper", padding: 0 }}
+        onClick={() => onPreviewClick()}
+      >
+        <Visibility />
```

Comment:
Use the correct icon (i think it's a smartphone, follow figma samples)

Reply:
[397b153](https://github.com/pagopa/io-services-cms/pull/810/commits/397b153fd561f5ba1feaae755c67e3dd64da15a4)